### DISCUSSION TITLE
jupyter-book + ablog - Links not created for date > 2021

### DISCUSSION THREAD

**Discussion by raphaeltimbo** [executablebooks/meta#277] [Q&A]
Hi there!
I have been trying to use jupyter-book and [ablog](https://ablog.readthedocs.io/manual/posting-and-listing/) in a project to create blog posts.
Everything was working ok, but now when I try to create a new blog post with:

```
'''{post} 2021-12-12
:author: Raphael Timbó
:tags: fórum 
'''
```

the link and excerpt for this post does not appear in the blog page.
This only happens if the date is > 2020. 
Does anyone know why this is happening?

Thanks!

**Comment by choldgraf** [comment]
Could it be because your date is in the future? I believe that ABlog only makes posts once the date of the post has passed. Maybe that's what is going on
